### ISSUE TITLE
Missing module     lava/src/lava/utils/plots.py

### ISSUE THREAD

**Issue by adimitr** [conda-forge/lava-feedstock#14]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

When I installed lava in a miniconda environment, plots.py was not there, and correspondingly some tutorials were not working. I dropped plots.py from the github source and things are fine now, but bringing it up so it can work out of the box.

### Installed packages

```shell
# packages in environment at /home/alex/miniconda3/envs/lava_nc:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main  
_openmp_mutex             5.1                       1_gnu  
aiofiles                  22.1.0          py311h06a4308_0  
aiosqlite                 0.18.0          py311h06a4308_0  
anyio                     3.5.0           py311h06a4308_0  
argon2-cffi               21.3.0             pyhd3eb1b0_0  
argon2-cffi-bindings      21.2.0          py311h5eee18b_0  
asteval                   0.9.31             pyhd8ed1ab_0    conda-forge
asttokens                 2.0.5              pyhd3eb1b0_0  
attrs                     23.1.0          py311h06a4308_0  
babel                     2.11.0          py311h06a4308_0  
backcall                  0.2.0              pyhd3eb1b0_0  
beautifulsoup4            4.12.2          py311h06a4308_0  
blas                      1.0                         mkl  
bleach                    4.1.0              pyhd3eb1b0_0  
brotli                    1.0.9                h5eee18b_7  
brotli-bin                1.0.9                h5eee18b_7  
brotli-python             1.0.9           py311h6a678d5_7  
bzip2                     1.0.8                h7b6447c_0  
ca-certificates           2023.08.22           h06a4308_0  
certifi                   2023.7.22       py311h06a4308_0  
cffi                      1.15.1          py311h5eee18b_3  
charset-normalizer        2.0.4              pyhd3eb1b0_0  
comm                      0.1.2           py311h06a4308_0  
contourpy                 1.0.5           py311hdb19cb5_0  
cryptography              41.0.3          py311hdda0065_0  
cycler                    0.11.0             pyhd3eb1b0_0  
cyrus-sasl                2.1.28               h52b45da_1  
dbus                      1.13.18              hb2f20db_0  
debugpy                   1.6.7           py311h6a678d5_0  
decorator                 5.1.1              pyhd3eb1b0_0  
defusedxml                0.7.1              pyhd3eb1b0_0  
entrypoints               0.4             py311h06a4308_0  
executing                 0.8.3              pyhd3eb1b0_0  
expat                     2.5.0                h6a678d5_0  
fontconfig                2.14.1               h4c34cd2_2  
fonttools                 4.25.0             pyhd3eb1b0_0  
freetype                  2.12.1               h4a9f257_0  
giflib                    5.2.1                h5eee18b_3  
glib                      2.69.1               he621ea3_2  
gst-plugins-base          1.14.1               h6a678d5_1  
gstreamer                 1.14.1               h5eee18b_1  
icu                       73.1                 h6a678d5_0  
idna                      3.4             py311h06a4308_0  
intel-openmp              2023.1.0         hdb19cb5_46306  
ipykernel                 6.25.0          py311h92b7b1e_0  
ipython                   8.15.0          py311h06a4308_0  
ipython_genutils          0.2.0              pyhd3eb1b0_1  
jedi                      0.18.1          py311h06a4308_1  
jinja2                    3.1.2           py311h06a4308_0  
jpeg                      9e                   h5eee18b_1  
json5                     0.9.6              pyhd3eb1b0_0  
jsonschema                4.19.2          py311h06a4308_0  
jsonschema-specifications 2023.7.1        py311h06a4308_0  
jupyter_client            8.6.0           py311h06a4308_0  
jupyter_core              5.5.0           py311h06a4308_0  
jupyter_events            0.8.0           py311h06a4308_0  
jupyter_server            2.10.0          py311h06a4308_0  
jupyter_server_fileid     0.9.0           py311h06a4308_0  
jupyter_server_terminals  0.4.4           py311h06a4308_1  
jupyter_server_ydoc       0.8.0           py311h06a4308_1  
jupyter_ydoc              0.2.4           py311h06a4308_0  
jupyterlab                3.6.3           py311h06a4308_0  
jupyterlab_pygments       0.1.2                      py_0  
jupyterlab_server         2.25.1          py311h06a4308_0  
kiwisolver                1.4.4           py311h6a678d5_0  
krb5                      1.20.1               h143b758_1  
lava                      0.5.0              pyhd8ed1ab_0    conda-forge
lcms2                     2.12                 h3be6417_0  
ld_impl_linux-64          2.38                 h1181459_1  
lerc                      3.0                  h295c915_0  
libbrotlicommon           1.0.9                h5eee18b_7  
libbrotlidec              1.0.9                h5eee18b_7  
libbrotlienc              1.0.9                h5eee18b_7  
libclang                  14.0.6          default_hc6dbbc7_1  
libclang13                14.0.6          default_he11475f_1  
libcups                   2.4.2                h2d74bed_1  
libdeflate                1.17                 h5eee18b_1  
libedit                   3.1.20221030         h5eee18b_0  
libffi                    3.4.4                h6a678d5_0  
libgcc-ng                 11.2.0               h1234567_1  
libgfortran-ng            11.2.0               h00389a5_1  
libgfortran5              11.2.0               h1234567_1  
libgomp                   11.2.0               h1234567_1  
libllvm14                 14.0.6               hdb19cb5_3  
libpng                    1.6.39               h5eee18b_0  
libpq                     12.15                hdbd6064_1  
libsodium                 1.0.18               h7b6447c_0  
libstdcxx-ng              11.2.0               h1234567_1  
libtiff                   4.5.1                h6a678d5_0  
libuuid                   1.41.5               h5eee18b_0  
libwebp                   1.3.2                h11a3e52_0  
libwebp-base              1.3.2                h5eee18b_0  
libxcb                    1.15                 h7f8727e_0  
libxkbcommon              1.0.1                h5eee18b_1  
libxml2                   2.10.4               hf1b16e4_1  
lz4-c                     1.9.4                h6a678d5_0  
markupsafe                2.1.1           py311h5eee18b_0  
matplotlib                3.8.0           py311h06a4308_0  
matplotlib-base           3.8.0           py311ha02d727_0  
matplotlib-inline         0.1.6           py311h06a4308_0  
mistune                   2.0.4           py311h06a4308_0  
mkl                       2023.1.0         h213fc3f_46344  
mkl-service               2.4.0           py311h5eee18b_1  
mkl_fft                   1.3.8           py311h5eee18b_0  
mkl_random                1.2.4           py311hdb19cb5_0  
munkres                   1.1.4                      py_0  
mysql                     5.7.24               h721c034_2  
nbclassic                 1.0.0           py311h06a4308_0  
nbclient                  0.8.0           py311h06a4308_0  
nbconvert                 7.10.0          py311h06a4308_0  
nbconvert-core            7.10.0             pyhd8ed1ab_0    conda-forge
nbformat                  5.9.2           py311h06a4308_0  
ncurses                   6.4                  h6a678d5_0  
nest-asyncio              1.5.6           py311h06a4308_0  
networkx                  3.1             py311h06a4308_0  
notebook                  6.5.4              pyha770c72_0    conda-forge
notebook-shim             0.2.3           py311h06a4308_0  
numpy                     1.26.0          py311h08b1b3b_0  
numpy-base                1.26.0          py311hf175353_0  
openjpeg                  2.4.0                h3ad879b_0  
openssl                   3.0.12               h7f8727e_0  
overrides                 7.4.0           py311h06a4308_0  
packaging                 23.1            py311h06a4308_0  
pandocfilters             1.5.0              pyhd3eb1b0_0  
parso                     0.8.3              pyhd3eb1b0_0  
pcre                      8.45                 h295c915_0  
pexpect                   4.8.0              pyhd3eb1b0_3  
pickleshare               0.7.5           pyhd3eb1b0_1003  
pillow                    10.0.1          py311ha6cbd5a_0  
pip                       23.3            py311h06a4308_0  
platformdirs              3.10.0          py311h06a4308_0  
ply                       3.11            py311h06a4308_0  
prometheus_client         0.14.1          py311h06a4308_0  
prompt-toolkit            3.0.36          py311h06a4308_0  
psutil                    5.9.0           py311h5eee18b_0  
ptyprocess                0.7.0              pyhd3eb1b0_2  
pure_eval                 0.2.2              pyhd3eb1b0_0  
pycparser                 2.21               pyhd3eb1b0_0  
pygments                  2.15.1          py311h06a4308_1  
pyopenssl                 23.2.0          py311h06a4308_0  
pyparsing                 3.0.9           py311h06a4308_0  
pyqt                      5.15.10         py311h6a678d5_0  
pyqt5-sip                 12.13.0         py311h5eee18b_0  
pysocks                   1.7.1           py311h06a4308_0  
python                    3.11.5               h955ad1f_0  
python-dateutil           2.8.2              pyhd3eb1b0_0  
python-fastjsonschema     2.16.2          py311h06a4308_0  
python-json-logger        2.0.7           py311h06a4308_0  
pytz                      2023.3.post1    py311h06a4308_0  
pyyaml                    6.0.1           py311h5eee18b_0  
pyzmq                     25.1.0          py311h6a678d5_0  
qt-main                   5.15.2              h53bd1ea_10  
readline                  8.2                  h5eee18b_0  
referencing               0.30.2          py311h06a4308_0  
requests                  2.31.0          py311h06a4308_0  
rfc3339-validator         0.1.4           py311h06a4308_0  
rfc3986-validator         0.1.1           py311h06a4308_0  
rpds-py                   0.10.6          py311hb02cf49_0  
scipy                     1.11.3          py311h08b1b3b_0  
send2trash                1.8.2           py311h06a4308_0  
setuptools                68.0.0          py311h06a4308_0  
setuptools-scm            7.1.0           py311h06a4308_0  
sip                       6.7.12          py311h6a678d5_0  
six                       1.16.0             pyhd3eb1b0_1  
sniffio                   1.2.0           py311h06a4308_1  
soupsieve                 2.5             py311h06a4308_0  
sqlite                    3.41.2               h5eee18b_0  
stack_data                0.2.0              pyhd3eb1b0_0  
tbb                       2021.8.0             hdb19cb5_0  
terminado                 0.17.1          py311h06a4308_0  
tinycss2                  1.2.1           py311h06a4308_0  
tk                        8.6.12               h1ccaba5_0  
tornado                   6.3.3           py311h5eee18b_0  
traitlets                 5.7.1           py311h06a4308_0  
typing-extensions         4.7.1           py311h06a4308_0  
typing_extensions         4.7.1           py311h06a4308_0  
tzdata                    2023c                h04d1e81_0  
urllib3                   1.26.18         py311h06a4308_0  
wcwidth                   0.2.5              pyhd3eb1b0_0  
webencodings              0.5.1           py311h06a4308_1  
websocket-client          0.58.0          py311h06a4308_4  
wheel                     0.41.2          py311h06a4308_0  
xz                        5.4.2                h5eee18b_0  
y-py                      0.5.9           py311h52d8a92_0  
yaml                      0.2.5                h7b6447c_0  
ypy-websocket             0.8.2           py311h06a4308_0  
zeromq                    4.3.4                h2531618_0  
zlib                      1.2.13               h5eee18b_0  
zstd                      1.5.5                hc292b87_0
```

### Environment info

```shell
active environment : lava_nc
    active env location : /home/alex/miniconda3/envs/lava_nc
            shell level : 2
       user config file : /home/alex/.condarc
 populated config files : /home/alex/.condarc
          conda version : 23.10.0
    conda-build version : not installed
         python version : 3.11.5.final.0
       virtual packages : __archspec=1=skylake
                          __glibc=2.35=0
                          __linux=5.15.0=0
                          __unix=0=0
       base environment : /home/alex/miniconda3  (writable)
      conda av data dir : /home/alex/miniconda3/etc/conda
  conda av metadata url : None
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : /home/alex/miniconda3/pkgs
                          /home/alex/.conda/pkgs
       envs directories : /home/alex/miniconda3/envs
                          /home/alex/.conda/envs
               platform : linux-64
             user-agent : conda/23.10.0 requests/2.31.0 CPython/3.11.5 Linux/5.15.0-88-generic ubuntu/22.04.3 glibc/2.35 solver/libmamba conda-libmamba-solver/23.11.0 libmambapy/1.5.3
                UID:GID : 1001:1001
             netrc file : None
           offline mode : False
```

**Comment by Tobias-Fischer** [comment]
Hi @adimitr - is there a reason why you use a very old lava version? The plots.py is included in lava 0.8.0. Please upgrade your lava version and you will be good :).

**Comment by adimitr** [comment]
No reason, but that is what conda installed from conda-forge. I tried an
update yesterday and it said all is up to date.

I can of course use pip, but thought to give the conda way a try.
